### DISCUSSION TITLE
Render

### DISCUSSION THREAD

**Discussion by D3Pheonix** [cleolibrary/CLEO-Redux#15] [Q&A]
Hi, what about rendering, like in sampfuncs ?
`0B6F: render font 0@ draw_text "brr" pos 2@ 3@ color 4@`

...And waiting for someone will bring support for samp, like sampfuncs or moonloader (lua), a lot of people would start writing js scripts. But it is difficult for most to do it manually, without mods like these.

**Comment by x87** [comment]
I will work on SDK similar to the one CLEO 4 has. It would allow to implement custom commands.

**Comment by x87** [comment]
https://re.cleo.li/docs/en/using-sdk.html
